stdlib/public/TensorFlow/Parameterized.swift

```diff
@@ -71,18 +72,18 @@ public extension Parameterized where Parameters : ParameterGroup {
 //===----------------------------------------------------------------------===//
 
 // Tensors conform to the protocols by "identity".
-extension Tensor : Parameterized {
-  public typealias Parameters = Tensor<Scalar>
-
-  public var allParameters: Parameters {
-    get {
-      return self
-    }
-    set {
-      self = newValue
-    }
-  }
-}
+//extension Tensor : Parameterized {
+//  public typealias Parameters = Tensor<Scalar>
+//
+//  public var allParameters: Parameters {
+//    get {
+//      return self
+//    }
+//    set {
+//      self = newValue
+//    }
+//  }
+//}
```

Comment:
I think removing this makes sense. It's fine for `Tensor` to not conform to `Parameterized` and to act as a "base case" member.